packages/react-i18n/docs/react_i18n_schema.md

```diff
@@ -0,0 +1,295 @@
+# Introduction <!-- omit in toc -->
```

Comment:
Seems to omit the section from the ToC:
![image](https://user-images.githubusercontent.com/50832/94567760-cebfcd80-0239-11eb-9029-cb81ebff059b.png)

Reply:
Yes, that is the syntax that [Markdown All In One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) uses to prevent the heading from getting added to the auto-generated table of content.